scripts/windows/clean-disk-dism.ps1

```
@@ -60,6 +60,8 @@ net stop wuauserv
 Set-Service wuauserv -StartupType Disabled
 
 # Clean up files (including those not addressed by cleanmgr)
+# Disable error checking as this sometimes throws errors that can be safely ignored (mostly locked files)
```

You can use try { } catch { } to ignore errors here as well.  Not 100% if that's PS 2 compatible